src/plot_butterfly.jl

```diff
@@ -131,25 +128,17 @@ function plot_butterfly!(
     f_grid = f[1, 1]
 
     if (topolegend)
-        topoAxis = Axis(
-            f_grid,
-            width = Relative(topowidth),
-            height = Relative(topoheight),
-            halign = topohalign,
-            valign = topovalign,
-            aspect = topoaspect,
-        )
+        topo_axis = update_axis(indiv_docstrings(:topo_default); topo_axis...)
```

Comment:
ah okay, here I see it; but then the function name is weird, no? If you want to define the defaults here, I suggest you just use PlotConfig(:topo_default) or something - no?

Reply:
I don't need PlotConfig here because i don't need multiple fields
 That is how it look like:
![2024-09-23_11-02-48](https://github.com/user-attachments/assets/e71b7916-2488-445c-bfb1-521c0e4f7109)

Reply:
I called it indiv_docstrings() because it add individual axis field in docstring
Could be renamed to: edit_docstring, axis_defaults, supportive_axis_defaults
Which do you prefer?

Reply:
ok I understand a little bit - you dont need a full PlotConfig, but just for e.g. the axis or something - but someone else will likely not catch that sublety - why are some defaults in PlotConfig fields, others not. renaming is highly recommended, plot_config_supportive or something maybe? still a bad name, plot_config_supportive_defaults :-D ok dont really know

Reply:
fine with me, then add in the docstring a description to the relation to PlotConfig please :)

Reply:
and maybe move them to PlotConfig so all defaults are in one file